services/wfm/src/main/java/org/openkilda/wfm/topology/AbstractTopologyConfig.java

```diff
@@ -35,6 +35,9 @@
     @Key("parallelism")
     int getParallelism();
 
+    @Key("parallelism.new")
+    int getNewParallelism();
```

Comment:
AbstractTopologyConfig is the configuration base shared by multiple topologies, but "parallelism.new" means nothing to others except CacheTopology. Why can't it be in CacheTopologyConfig? 
BTW, What does "new" in parallelism.new means?

Reply:
As I understand, we have all bolts with parallelism = 1. But all new bolts in Storm must have parallelism at least 2. I think that it's applicable for all topologies, that's why I put parameter in that general class. So, new bolts will have new parallelism. Maybe we must rename it somehow.